File: HardwareInfoCollector/models/hardware.py
class Hardware:
    ProductType: int
    Model: str
    Price: int
    Link: str

    def __init__(self, product_type: int, model: str, price: int, link: str):
        self.ProductType = product_type
        self.Model = model
        self.Price = price
        self.Link = link

File: BuildGenerator/src/Bll/Models/Hardware/Abstractions/Hardware.cs
namespace Bll.Models.Hardware.Abstractions;

public abstract class Hardware
{ 
    /// <summary>
    /// Price of the hardware.
    /// </summary>
    double Price { get; set; }
    
    /// <summary>
    /// Link from the E-market of the hardware.
    /// </summary>
    string? Link { get; set; }

    /// <summary>
    /// Model of the hardware.
    /// </summary>
    string? Model { get; set; }
}